### PR TITLE
logging out button for researchers and students

### PR DESCRIPTION
**Pull request by rosasck**
Logs you out and if you try to be sneaky and use the back arrow it takes you to login page :) 

-- The user page being removed is on another PR so that will be gone after that PR :)

### CODE REVIEW

File: simcon_project/simcon_project/urls.py [hunk 1]
```diff
@@ -27,26 +27,37 @@
...
     path('student/', student_view, name="student-view"),
-    path('student/settings/', student_settings_view, name="student-settings-view"),
-    path('student/register/<uidb64>/', student_registration, name="student-registration"),
-    path('student/conversation/', include('conversation_templates.urls.conv_urls'), name='conversation'),
+    path('student/settings/', student_settings_view,
+         name="student-settings-view"),
+    path('student/register/<uidb64>/',
+         student_registration, name="student-registration"),
+    path('student/conversation/',
+         include('conversation_templates.urls.conv_urls'), name='conversation'),
```
Comment (Mkay971) [inline simcon_project/simcon_project/urls.py]: your formatter again.

**Review by Mkay971 — DISMISSED** [review]
Perfect.

File: simcon_project/simcon_project/settings.py [hunk 2]
```diff
@@ -140,7 +140,7 @@
 STATIC_URL = '/static/'
 MEDIA_URL = '/media/'
 LOGIN_REDIRECT_URL = 'redirect-from-login'
-LOGOUT_REDIRECT_URL = 'login'
+LOGOUT_REDIRECT_URL = '/'
```
Comment (ericharper1) [inline simcon_project/simcon_project/settings.py]: Hmm, I kind of like logout going to the login screen...

Other opinions?

File: simcon_project/simcon_project/urls.py [hunk 3]
```diff
@@ -47,6 +48,6 @@
     path('researcher/students/<str:name>/', student_management, name="student-management"),
     path('researcher/students/', student_management, name="student-management"),
     path('researcher/response/', view_response, name="view-response"),
-    path('researcher/response/update/<uuid:pk>/', update_overall_response_feedback, name='update-overall-response-feedback'),
```
Comment (ericharper1) [inline simcon_project/simcon_project/urls.py]: lol, was it one space away from the line being too long?

File: simcon_project/templates/researcher_base.html [hunk 4]
```diff
@@ -23,6 +23,7 @@
         </ul>
         <ul class="nav navbar-nav navbar-right">
           <li>{% block nav-userView %}<a href="{% url 'researcher-user-view' %}"><span class="glyphicon glyphicon-user"></span> User</a>{% endblock %}</li>
+          <li>{% block nav-logoutView %}<a href="{% url 'logout' %}"><span class="	glyphicon glyphicon-log-out"></span> Log Out</a>{% endblock %}</li>
```
Comment (ericharper1) [inline simcon_project/templates/researcher_base.html]: I'd like logout to be the right-most thing on the nav-bar. 

Other opinions?

File: simcon_project/templates/student_base.html [hunk 5]
```diff
@@ -21,6 +21,7 @@
         </ul>
         <ul class="nav navbar-nav navbar-right">
           <li>{% block nav-userView %}<a href="{% url 'student-user-view' %}"><span class="glyphicon glyphicon-user"></span> User</a>{% endblock %}</li>
+          <li>{% block nav-logoutView %}<a href="{% url 'logout' %}"><span class="	glyphicon glyphicon-log-out"></span> Log Out</a>{% endblock %}</li>
```
Comment (ericharper1) [inline simcon_project/templates/student_base.html]: Same here.

**Review by ericharper1 — COMMENTED** [review]
Works great! Left some comments about stylistic preference. Hopefully we can get some other folks' opinions in here.

File: simcon_project/simcon_project/settings.py (hunk 2 above)
Comment (kaigoos) [reply simcon_project/simcon_project/settings.py]: I think that's a good idea too

Comment (rosasck) [reply simcon_project/simcon_project/settings.py]: wasnt sure where to direct it but will fix!

File: simcon_project/simcon_project/urls.py (hunk 3 above)
Comment (rosasck) [reply simcon_project/simcon_project/urls.py]: auto formatter mess

File: simcon_project/templates/researcher_base.html (hunk 4 above)
Comment (rosasck) [reply simcon_project/templates/researcher_base.html]: Will Adjust!

File: simcon_project/templates/student_base.html (hunk 5 above)
Comment (rosasck) [reply simcon_project/templates/student_base.html]: ditto!

**Review by kaigoos — DISMISSED** [review]
Pressing back after logging out isn't taking me back to the login screen, but will actually take me back to the page I was just on. When I logged in as the researcher and went to the  template management screen then logged out, I was able to press back and go back to template management and also delete a template from there. After the post request went through I was redirected back to login screen. 
Also if I press back after the login screen and then press a link or a button it will redirect me to the login screen, but there will be a query string after "/accounts/login/" and it will take me to a different page than the home page after logging in.

**Comment by rosasck** [comment]
@kaigoos  its currently not supposed to i will fix it after work! it currently should take u to a "thank you for using sim con" page. As for issues for login, im not sure. that would be @ericharper1, im only working on the logout portion

File: simcon_project/simcon_project/urls.py [hunk 6]
```diff
@@ -34,19 +34,29 @@
 
     # Stuff students can see
     path('student/', student_view, name="student-view"),
-    path('student/settings/', student_settings_view, name="student-settings-view"),
-    path('student/register/<uidb64>/', student_registration, name="student-registration"),
-    path('student/conversation/', include('conversation_templates.urls.conv_urls'), name='conversation'),
+    path('student/settings/', student_settings_view,
+         name="student-settings-view"),
```
Comment (ericharper1) [inline simcon_project/simcon_project/urls.py]: Looks like your auto-formatter got a hold of this file lol

**Review by ericharper1 — CHANGES_REQUESTED** [review]
Revert auto format in urls :)

File: simcon_project/simcon_project/urls.py (hunk 6 above)
Comment (rosasck) [reply simcon_project/simcon_project/urls.py]: ...how.. i didnt even touch it

**Review by kaigoos — APPROVED** [review]

**Review by ericharper1 — APPROVED** [review]

**Pull request merged**